subprojects/ide-native/src/main/java/org/gradle/ide/visualstudio/VisualStudioProject.java

```diff
@@ -44,10 +44,6 @@
  */
 @Incubating
 public interface VisualStudioProject extends BuildableComponentSpec {
```

Comment:
Probably.  I think this was there more to get a ready-to-use Named, Buildable implementation.  I'm not sure how else it might have been used.  I've removed it and implemented Named, Buildable instead.